### PR TITLE
[WIP] src/v2/guide/typescript.md

### PR DESCRIPTION
**Pull request by tamnd**
TODO

- [x] Translate `Official declaration in NPM packages`
- [x] Translate `Recommended configuration`
- [ ] Translate `Using Vue's type declarations`
- [ ] Translate `Class-style vue components`
- [ ] Translate `Declaring types of Vue plugins`

### CODE REVIEW

File: src/v2/guide/typescript.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+Chúng tôi có kế hoạch chuyển tất cả các khai báo (declaration) sang export kiểu ES trong tương lai. Vì thế hãy sử dụng cấu hình được đề xuất dưới đây cho các ứng dụng tương lai của bạn.
 
-## Official Declaration in NPM Packages
+## Khai báo trong các gói NPM (NPM package)
 
-A static type system can help prevent many potential runtime errors, especially as applications grow. That's why Vue ships with [official type declarations](https://github.com/vuejs/vue/tree/dev/types) for [TypeScript](https://www.typescriptlang.org/) - not only in Vue core, but also for [vue-router](https://github.com/vuejs/vue-router/tree/dev/types) and [vuex](https://github.com/vuejs/vuex/tree/dev/types) as well.
+Một hệ thống kiểu tĩnh có thể giúp phát hiện được nhiều lỗi thời gian chạy (runtime error), đặc biệt khi ứng dụng trở lên lớn. Đó cũng là lý do tại sao Vue đi kèm với [khai báo kiểu]() cho [Typescript](https://typescriptlang.org/) - không chỉ core Vue, mà cả [vue-router](https://github.com/vuejs/vue-router/tree/dev/types) và [vuex](https://github.com/vuejs/vuex/tree/dev/types).
```
Comment (ldhnam) [inline src/v2/guide/typescript.md:15]: ứng dụng trở lên lớn -> ứng dụng trở nên lớn

File: src/v2/guide/typescript.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+Một hệ thống kiểu tĩnh có thể giúp phát hiện được nhiều lỗi thời gian chạy (runtime error), đặc biệt khi ứng dụng trở lên lớn. Đó cũng là lý do tại sao Vue đi kèm với [khai báo kiểu]() cho [Typescript](https://typescriptlang.org/) - không chỉ core Vue, mà cả [vue-router](https://github.com/vuejs/vue-router/tree/dev/types) và [vuex](https://github.com/vuejs/vuex/tree/dev/types).  
 
-Since these are [published on NPM](https://cdn.jsdelivr.net/npm/vue/types/), and the latest TypeScript knows how to resolve type declarations in NPM packages, this means when installed via NPM, you don't need any additional tooling to use TypeScript with Vue.
+Vì các gói được phát hành trên NPM](), và phiên bản mới nhất của Typescript hiểu dược các khai báo kiểu trong các gói NPM, nên khi cài đặt Vue bằng NPM, bạn không cần phải cài thêm bất kì thứ gì khác để có thể sử dụng Typescript với Vue.
```
Comment (ldhnam) [inline src/v2/guide/typescript.md:17]: hiểu dược -> hiểu được

**Comment by namdau** [comment]
Thank you for your contribution but unfortunately we'll gonna archive this repo in favor of the official v3 documentation which is being forked here https://github.com/vuejs-vn/docs-next.

Please help to submit your PR over there from now on.

**Comment by namdau** [comment]
Close this one in favor of adapting [v3 doc repo ](https://github.com/vuejs-vn/docs-next).

**Comment by namdau** [comment]
Close this one in favor of adapting [v3 doc repo ](https://github.com/vuejs-vn/docs-next).

**Pull request closed without merging**